superheroes.yaml

```
@@ -0,0 +1,510 @@
+
+# rest-fights-native(): podman run -d -p 8084:8084 --name rest-fights-native -e QUARKUS_MONGODB_HOSTS=localhost:27017 -e QUARKUS_LIQUIBASE_MONGODB_MIGRATE_AT_START="false" -e QUARKUS_MONGODB_CREDENTIALS_USERNAME=superfight -e QUARKUS_MONGODB_CREDENTIALS_PASSWORD=superfight -e QUARKUS_STORK_HERO_SERVICE_SERVICE_DISCOVERY_ADDRESS_LIST=localhost:8083 -e QUARKUS_STORK_VILLAIN_SERVICE_SERVICE_DISCOVERY_ADDRESS_LIST=localhost:8084 -e MP_MESSAGING_CONNECTOR_SMALLRYE_KAFKA_APICURIO_REGISTRY_URL="http://localhost:8086/apis/registry/v2" -e KAFKA_BOOTSTRAP_SERVERS=PLAINTEXT://localhost:9092 --network host --replace quay.io/quarkus-super-heroes/rest-fights:native-latest
+
+# (optional) ui: podman run -d -p 8080:8080 --name ui-super-heroes -e API_BASE_URL=http://localhost:8082 --network host --replace quay.io/quarkus-super-heroes/ui-super-heroes:native-latest
+
+scripts:
+
+  infer-datastouce-hostnames:
+  # infer the hostnames based on the provided host servers
+  - set-state: HEROES_DB_HOSTNAME ${{= getHostname( '${{DS_SERVER}}' )}}
+  - set-state: VILLAINS_DB_HOSTNAME ${{= getHostname( '${{DS_SERVER}}' )}}
+  - set-state: FIGHTS_DB_HOSTNAME ${{= getHostname( '${{DS_SERVER}}' )}}
+  - set-state: APICURIO_HOSTNAME ${{= getHostname( '${{DS_SERVER}}' )}}
+  - set-state: KAFKA_HOSTNAME ${{= getHostname( '${{DS_SERVER}}' )}}
+  - set-state: OTEL_HOSTNAME ${{= getHostname( '${{DS_SERVER}}' )}}
+
+  infer-services-hostnames:
+  # infer the hostnames based on the provided host servers
+  - set-state: HEROES_REST_HOSTNAME "${{= getHostname( '${{SUT_SERVER}}' )}}"
+  - set-state: VILLAINS_REST_HOSTNAME "${{= getHostname( '${{SUT_SERVER}}' )}}"
+  - set-state: LOCATIONS_GRPC_HOSTNAME "${{= getHostname( '${{SUT_SERVER}}' )}}"
+  - set-state: FIGHTS_REST_HOSTNAME "${{= getHostname( '${{SUT_SERVER}}' )}}"
+
+  prepare-superheroes:
+  # clone the repository and checkout the proper commit/branch
+  - sh: if [ -d ${{SUPERHEROES_FOLDER}} ]; then rm -rf ${{SUPERHEROES_FOLDER}}; fi 
+  - sh: git clone --depth 1 ${{SUPERHEROES_REPO}} ${{SUPERHEROES_FOLDER}}
+  - sh: cd ${{SUPERHEROES_FOLDER}} && git fetch --depth 1 origin ${{SUPERHEROES_COMMIT:main}} && git checkout ${{SUPERHEROES_COMMIT:main}}
+
+  wait-for-images:
+  - wait-for: SUPERHEROES_IMAGES_READY
+
+  ###### OPENTELEMETRY ######
+  start-jaeger:
+  - log: starting Jaeger...
+  - sh: >
+      ${{PODMAN}} run -d
+      --name jaeger
+      -p 16686:16686
+      -p 14268:14268
+      -p 14250:14250
+      -p ${{OTEL_RECEIVER_PORT}}
+      ${{JAEGER_IMAGE:'docker.io/jaegertracing/all-in-one:1'}}
+    then:
+    - regex: (?<HOST.JAEGER_POD_ID>[a-f0-9]{64})
```

We only want the last id with 64 chars, otherwise it will match the first string that matches this pattern. If the image has not been pulled down, this will match the image ID, instead of the container ID

```suggestion
    - regex: (?<HOST.JAEGER_POD_ID>[a-f0-9]{64}$)
```